Packs/Anomali_ThreatStream/Integrations/AnomaliThreatStreamv3/AnomaliThreatStreamv3.py

```diff
@@ -545,6 +777,880 @@ def get_ip_reputation(client: Client, score_calc: DBotScoreCalculator, ip, statu
+        limit (Str): The maximum number of results to return.
+        page (Str): The page number of the results to retrieve.
+        page_size (Str):  The maximum number of objects to retrieve per page.
+    Returns:
+        (CommandResults).
+    """
+    params = return_params_of_pagination_or_limit(arg_to_number(page), arg_to_number(page_size), arg_to_number(limit))
+    if status_in == 'Processing':
+        params['status'] = 'processing'
+    elif status_in == 'Rejected':
+        params['status'] = 'deleted'
+    elif status_in == 'Ready To Review':
+        params['status'] = 'done'
+    elif status_in == 'Errors':
+        params['status'] = 'errors'
+    elif status_in == 'Approved':
+        params['status'] = 'approved'
+    res = client.list_import_job_request(import_id, params)
+    outputs = res.get("objects", res)
+    readable_output = tableToMarkdown("Import entries", outputs, removeNull=True, headerTransform=header_transformer,
+                                      headers=["id", "date", "status", "approved_by", "email", "intelligence_initiatives",
+                                               "numIndicators", "numRejected", "tags"],
+                                      json_transform_mapping={
+                                          'approved_by': JsonTransformer(keys=['email'], func=lambda hdr: hdr.get('email', '')),
+                                          'tags': JsonTransformer(func=lambda hdr: ", ".join([item.get('name') for item in hdr])),
+                                          'intelligence_initiatives': JsonTransformer(func=lambda hdr:
+                                                                                      ", ".join([item.get('type')
+                                                                                                for item in hdr]))})
+    if not import_id:
+        for item in outputs:
+            item['ImportID'] = item.pop('id')
+            item['JobID'] = item.pop('jobID')
+    else:
+        outputs['ImportID'] = outputs.pop('id')
+        outputs['JobID'] = outputs.pop('jobID')
+    return CommandResults(
+        outputs_prefix=f"{THREAT_STREAM}.Import",
+        outputs_key_field="ImportID",
+        ignore_auto_extract=True,
+        outputs=outputs,
+        readable_output=readable_output,
+        raw_response=res,
+    )
+
+
+def approve_import_job_command(client: Client, import_id: str = None) -> CommandResults:
+    """
+    Approving all observables in an import job
+    Args:
+        client (Client): Client to perform calls to Anomali ThreatStream service.
+        import_id (Str):  The id of a specific import entry.
+    Returns:
+        (CommandResults).
+    """
+    res = client.approve_import_job_request(import_id)
+    if res.get("approved_by_id"):
+        readable_output = 'The import session was successfully approved.'
+    else:
+        raise DemistoException('Import Session Approval Failed.')
```

Comment:
What returns from the API?

Reply:
When the import cant be approved (error status) the "approved_by_id" value is null